### ISSUE TITLE
Settings: announce deleted custom font

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/web-stories-wp#11149]
## Feature Description

On the settings page, when removing a custom font we should announce that using the `speak()` / `useLiveRegion` helper.
